engine/pokedex/pokedex.asm

```diff
@@ -1492,7 +1492,7 @@ Pokedex_PrintListing:
 .loop
 	push af
 	ld a, [de]
-	ld [wd265], a
+	ld [wTempSpecies], a
```

Comment:
This one is used for `GetPokemonName` later in the code, which of course uses `wNamedObjectIndexBuffer`.
I know `wTempSpecies` is used around the entire pokedex for the purpose you've named it, and changing it in only some places would be confusing, but it's also confusing where `wNamedObjectIndexBuffer` for `GetPokemonName` is set, if you don't know both are the same address.

Reply:
This is one of those cases that can only be solved by adding the extra information in a comment.
Maybe an assert? We could borrow Prism's assert macro and do `assert wNamedObjectIndexBuffer == wTempSpecies` right above?

Unless this address isn't read again by the dex code, in which case there's no need for an assert as there wouldn't be any issues if the labels were given different addresses...